src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -84,6 +85,13 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             CurrentConfiguration = configurationProvider.GetConfiguration();
 
             SetBoundSolutionUIContext();
+
+            gitEventsMonitor.HeadChanged += GitEventsMonitor_HeadChanged;
+        }
+
+        private void GitEventsMonitor_HeadChanged(object sender, EventArgs e)
+        {
+            SolutionBindingUpdated?.Invoke(this, EventArgs.Empty);
```

Comment:
Surely this should be raising the `PreSolutionBindingUpdated` event too?

Reply:
I have realised that and fixed it later PR (stateful branch)